pkg/operator/ceph/version/version_test.go

```diff
@@ -222,8 +206,7 @@ func TestCephVersion_Unsupported(t *testing.T) {
 		fields fields
 		want   bool
 	}{
-		{"supported", fields{Major: 14, Minor: 2, Extra: 1, Build: 0}, false},
-		{"supported", fields{Major: 14, Minor: 2, Extra: 12, Build: 0}, false},
+		{"supported", fields{Major: 16, Minor: 2, Extra: 1, Build: 0}, false},
 		{"supported", fields{Major: 15, Minor: 2, Extra: 1, Build: 0}, false},
 		{"supported", fields{Major: 15, Minor: 2, Extra: 6, Build: 0}, false},
 		{"unsupported", fields{Major: 14, Minor: 2, Extra: 13, Build: 0}, true},
```

Comment:
Should we add v17 (quincy) here and verify it is unsupported?

Reply:
added